include/Popups/tpls/header.tpl

```diff
@@ -79,7 +79,7 @@ function clearAll() {
 <input type="submit" name="button" class="button" id="search_form_submit"
 	title="{$APP.LBL_SEARCH_BUTTON_TITLE}"
 	value="{$APP.LBL_SEARCH_BUTTON_LABEL}" />
-<input type="reset" onclick="SUGAR.searchForm.clear_form(this.form); return false;" class="button" id="search_form_clear"
+<input type="submit" onclick="SUGAR.searchForm.clear_form(this.form); return false;" class="button" id="search_form_clear"
```

Comment:
Yes it does but its still not correct - we must have a rest set css style somewhere.